docs/tutorial/accessibility.md

```diff
@@ -31,3 +31,32 @@ In Devtron, there is a new accessibility tab which will allow you to audit a pag
 Both of these tools are using the [Accessibility Developer Tools](https://github.com/GoogleChrome/accessibility-developer-tools) library built by Google for Chrome. You can learn more about the accessibility audit rules this library uses on that [repository's wiki](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules).
 
 If you know of other great accessibility tools for Electron, add them to the [accessibility documentation](https://electron.atom.io/docs/tutorial/accessibility) with a pull request.
+
+## Enabling Accessibility
+
+Electron applications keep accessibility disabled by default for performance reasons but there are multiple ways to enable it.
+
+### Inside Application
+
+By using [`app.setAccessibilitySupportEnabled(enabled)`](https://electron.atom.io/docs/api/app.md#appsetaccessibilitysupportenabledenabled-macos-windows), you can expose accessibility switch to users in the application preferences.
+
+### Assistive Technology
+
+Electron application will enable accessibility automatically when it detects assistive technology (Windows) or VoiceOver (macOS). See Chrome's [accessibility documentation](https://www.chromium.org/developers/design-documents/accessibility#TOC-How-Chrome-detects-the-presence-of-Assistive-Technology) for more details.
```

Comment:
"Electron application will enable accessibility automatically" -- just so I understand correctly: does this mean the presence of these assistive features will automatically cause accessibility to be enabled, regardless of the current setting?

Or does it only mean that these features will be used if `setAccessibilitySupportEnabled(true)` was explicitly called in the app?

Reply:
The first one – system assistive utilities have a priority and will enable accessibility regardless of the `setAccessibilitySupportEnabled()`. So an Electron app's developer can be unaware of this method and accessibility will still work for many users who have VoiceOver on.

For users who need accessibility without VoiceOver there are 3 options:
- ask an Electron app's developer to expose accessibility switch in settings using `setAccessibilitySupportEnabled(true)`
- ask their assistive software developer to start applying the `AXManualAccessibility` attribute on Electron apps
- use some custom utility to apply `AXManualAccessibility` on Electron apps (I'm going to create an open-source one later)